src/equations/shallow_water_1d.jl

```diff
@@ -47,14 +47,20 @@ References for the SWE are many but a good introduction is available in Chapter
 struct ShallowWaterEquations1D{RealT<:Real} <: AbstractShallowWaterEquations{1, 3}
```

Comment:
The docstring at the top of this file with
```julia
@doc raw"""
    ShallowWaterEquations1D(gravity, H0)
blah blah blah
```
should be updated to include the new keyword arguments
```julia
@doc raw"""
    ShallowWaterEquations1D(gravity, H0, threshold_limiter, threshold_wet)
blah blah blah
```
along with a short sentence about what these threholds are.